src/opentimelineio/track.cpp

```diff
@@ -196,7 +197,13 @@ std::map<Composable*, TimeRange> Track::range_of_all_children(ErrorStatus* error
                                       transition->out_offset() + transition->in_offset());
         }
         else if (auto item = dynamic_cast<Item*>(child.value)) {
-            auto last_range = TimeRange(last_end_time, item->trimmed_range(error_status).duration());
+            auto output_range = TimeRange(RationalTime(0), item->trimmed_range(error_status).duration());
```

Comment:
Since considerable work is potentially being done in the following `for` loop, should `error_status` be right after instead of later below?